pkg/scripts/os.go

```diff
@@ -55,13 +55,12 @@ EOF
 
 sudo mkdir -p /etc/apt/apt.conf.d
 cat <<EOF | sudo tee /etc/apt/apt.conf.d/proxy.conf
-{{- if .HTTPS_PROXY -}}
-Acquire::https::Proxy = "{{ .HTTPS_PROXY }}";
-{{ end }}
-
+{{- if .HTTPS_PROXY }}
+Acquire::https::Proxy "{{ .HTTPS_PROXY }}";
+{{ end -}}
 {{- if .HTTP_PROXY -}}
-Acquire::http::Proxy = "{{ .HTTP_PROXY }}";
-{{ end }}
+Acquire::http::Proxy "{{ .HTTP_PROXY }}";
+{{ end -}}
```

Comment:
this last `{{ end -}}` broke the rendering result (from bash PoV).
The rendering result is `cat <<EOF | sudo tee /etc/apt/apt.conf.d/proxy.confEOF` which caused kubeadm to be never installed and fail everything else.

Reply:
thanks for the feedback - i will fix it and push again after lunch - can i retrigger the tests myself?